src/Main.hs

```diff
@@ -50,6 +56,11 @@ data HtmlRoute
     via ( HtmlRoute
             `WithSubRoutes` [ FileRoute "index.html"
                             , FileRoute "about.html"
+                            , FileRoute "upcomingevents.html"
+                            , FileRoute "pastevents.html"
+                            , FileRoute "connectwithus.html"
+                            , FileRoute "fpjobsinindia.html"
+                            , FileRoute "resources.html"
```

Comment:
For shorter URLs, I'd suggest renaming these to:

- "upcoming.html"
- "past.html"
- "connect.html"
- "jobs.html"
- "resources.html"